src/sentry/static/sentry/app/views/alerts/rules/details/body.tsx

```diff
@@ -300,6 +302,19 @@ export default class DetailsBody extends React.Component<Props> {
                         filter={queryWithTypeFilter}
                       />
                     )}
+                    {rule?.dataset === Dataset.TRANSACTIONS && (
+                      <RelatedTransactions
+                        organization={organization}
+                        location={location}
+                        rule={rule}
+                        projects={((projects as Project[]) || []).filter(project =>
```

Comment:
we might not need to filter by the rule's projects as I try to get only the project slugs relevant to the alert rule. I know there were problems before with that but hopefully should be fixed by now.